Sources/Build/PkgConfig.swift

```diff
@@ -217,6 +213,44 @@ struct PkgConfigParser {
         return result
     }
     
+    /// Split line on unescaped spaces 
+    /// Will break on space in "abc def" and "abc\\ def" but not in "abc\ def" and ignore
+    /// multiple spaces such that "abc   def" will split into ["abc", "def"].
+    private func splitEscapingSpace(_ line: String) -> [String] {
+        let source = Array(line.characters)
+        var startIndex = 0
+        var index = 0
+        var splits = [String]()
+        var fragment = ""
+        
+        func saveFragment() {
+            if index - startIndex > 0 {
+                fragment += String(source[startIndex..<index])
+            }
+            if fragment.characters.count > 0 {
+                splits.append(fragment)
+            }
+        }
+        
+        while index < source.count {
```

Comment:
I would recommend writing this loop using an iterator over the characters... then you can advance the iterator if you see a "\\" to get the next character. My guess is this will end up being more succinct and readable... see "it.next" w/in this file for an example.